### PR TITLE
[Meta] Update readme

### PR DESCRIPTION
**Pull request by Razish**
Also passes markdownlint now :)
Best to view the rich diff.

Resolves #1135

## TODO

- [ ] Include some recommendations from #971

### CODE REVIEW

**Comment by ensiform** [comment]
Should make clarification that outcast MP isn't available (but check jk2mv repo) and that single player may be experimental perhaps?

**Comment by Razish** [comment]
> Should make clarification that outcast MP isn't available (but check jk2mv repo) and that single player may be experimental perhaps?

Yep, I have:
image

**Comment by AlexCS1337** [comment]
This is another project you can add under JK DF2 if you want (it also works for multiplayer) https://github.com/r1sc/OpenSith

File: README.md [hunk 1]
```diff
@@ -1,99 +1,106 @@
...
-<a href="https://discord.gg/dPNCfeQ"><img src="https://img.shields.io/badge/discord-join-7289DA.svg?logo=discord&longCache=true&style=flat" /></a>
-[![Forum](https://img.shields.io/badge/forum-JKHub.org%20OpenJK-brightgreen.svg)](https://jkhub.org/forums/forum/49-openjk/)
+## Supported Games
 
-[![Coverity Scan Build Status](https://scan.coverity.com/projects/1153/badge.svg)](https://scan.coverity.com/projects/1153)
+| Game | Single Player | Multi Player |
+| - | - | - |
+| Jedi Academy | ✅ Stable | ✅ Stable |
+| Jedi Outcast | 😧 Works, needs attention | 🙅 Not supported - consider [JK2MV](https://jk2mv.org) |
+| Jedi Knight DF2 | 💡 Consider [OpenJKDF2](https://github.com/shinyquagsire23/OpenJKDF2) | 😶 N/A |
```
Comment (AlexCS1337) [inline README.md:24]: This is another project you can add under JK DF2 if you want (it also works for multiplayer) https://github.com/r1sc/OpenSith

**Review by AlexCS1337 — APPROVED** [review]
i approve :p

**Comment by CansecoDev** [comment]
I'd suggest to add back (in a different section maybe) OpenJKDF2, even if not the same engine, it's from the same series and people might want to know about it's existence.

**Review by ensiform — APPROVED** [review]

**Pull request merged**